src/templates/chapter.js

```diff
@@ -400,6 +402,7 @@ const ChapterTemplate = ({ data: { mdx: chapter } }) => {
                     fontFamily: "'Inconsolata', monospace",
                     renderSideBySide: false,
                     wordWrap: true,
+                    renderIndicators: false,
```

Comment:
By default `renderIndicators` option is set to `true`, no need for explicitly defining it.